### DISCUSSION TITLE
Cannot get rtorrent to run python script

### DISCUSSION THREAD

**Discussion by Span24** [rakshasa/rtorrent#1201] [General]
I am trying to get rtorrent to run a python script after download completion.  No matter what I do the script is not running.  I am also unable to get any logging to help track the problem down.  I've tried debug logging and nothing related to the running of the script is found.  I can execute the script without issue from the console.  Here is the line im using to execute the script from rtorrent.rc:

system.method.set_key = event.download.finished,orpheus-better,"execute=/usr/share/orpheusbetter-crawler/orpheusbetter.py"

To TRY to troubleshoot I have a line in the .py code to create a simple .txt file.  When executed from the console the file is created and the expected processing of the script takes place.  From rtorrent, following successful download, I get nothing.

Any help to correct my system.method.set_key line or a way to get rtorrent to log what's going on would be greatly appreciated.

Thanks!

**Comment by kannibalox** [comment]
You can use [`log.execute`](https://rtorrent-docs.readthedocs.io/en/latest/cmd-ref.html#term-log-execute) to set a log file where rTorrent will print all of the execute.* commands and the return codes. It can grow pretty big, so it's best to only enable it while troubleshooting. Some simple things to check first:
* does the rTorrent user have access to the script?
* is the script marked as executable?
  * if it is, does the shebang at the beginning point to the correct python path?
